strawberry/subscriptions/protocols/graphql_transport_ws/handlers.py

```diff
@@ -134,39 +130,28 @@ async def handle_connection_init_timeout(self) -> None:
     async def handle_task_exception(self, error: Exception) -> None:  # pragma: no cover
         self.task_logger.exception("Exception in worker task", exc_info=error)
 
-    async def handle_message(self, message: dict) -> None:
+    async def handle_message(self, message: Message) -> None:
         try:
-            message_type = message.pop("type")
+            if message["type"] == "connection_init":
+                await self.handle_connection_init(message)
 
-            if message_type == ConnectionInitMessage.type:
-                await self.handle_connection_init(ConnectionInitMessage(**message))
+            elif message["type"] == "ping":
```

Comment:
Why are we switching from constants to strings here? I get that these strings are obvious, but my intuition tells me a constant suits this better

Reply:
I want the new TypedDicts to be the single source of truth for the shape of graphql-transport-ws messages. However, unlike with dataclassses, it's not possible to store a constant value in a TypedDict. The only option is to use `typing.Literal` and let the type checker enforce that `message["typed"]` must always match one of the literal message types. (which is what we do here).

Reply:
I think with types, plain strings are better 😊